app/src/main/java/com/example/themealsapp/di/NetworkModule.kt

```diff
@@ -49,15 +39,14 @@ class NetworkModule {
     }
 
     @Provides
-    fun providesHttpLoggingInterceptor(): HttpLoggingInterceptor {
+    fun providesOkHttpLoggingInterceptor(): HttpLoggingInterceptor {
         return HttpLoggingInterceptor().apply {
-            level = HttpLoggingInterceptor.Level.BODY
+            level = HttpLoggingInterceptor.Level.BODY // Change it to NONE when app is launched to production
```

Comment:
here you can use the debug flag from BUILDCONFIG to define the level

Reply:
okkkk